src/DDTrace/Integrations/SymfonyBundle.php

```diff
@@ -4,12 +4,14 @@
 
 use DDTrace\Encoders\Json;
 use DDTrace\Tags;
+use DDTrace\Types;
 use DDTrace\Tracer;
 use DDTrace\Transport\Http;
 use OpenTracing\GlobalTracer;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 use Symfony\Component\HttpKernel\HttpKernel;
 use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Bridge\Monolog\Processor\DebugProcessor;
```

Comment:
Looks like this isn't used?

Reply:
Oh right, this is a leftover from testing I did to figure out how to access route name.